src/Components/LandingPage/LandingPage.tsx

```diff
@@ -56,7 +56,7 @@ export const LandingPage = () => {
   const [selectedBlueprint, setSelectedBlueprint] = useState<
     string | undefined
   >();
-  const { data: blueprints, isLoading } = useGetBlueprintsQuery({});
+  const { data: blueprints, isLoading, refetch } = useGetBlueprintsQuery({});
```

Comment:
I am aware of the fact that there are multiple ways to refetch this query. @amirfefer what are your thoughts? We will be doing this even when creating blueprints, so I am happy to hear your opinion.

Reply:
@avitova, I think we should tag invalidation to refetch.
https://redux-toolkit.js.org/rtk-query/usage/automated-refetching
We will need to use `providesTags` and `invalidatesTags` in an enhanced endpoint. You can add the enhanced endpoint to this file:
https://github.com/RedHatInsights/image-builder-frontend/blob/main/src/store/enhancedImageBuilderApi.ts
@amirfefer and I have discussed this on Slack, he may already have gotten started on this so check with him first!

Reply:
@avitova, like react-query, redux's RTK has cache management for its data store, instead of manually refetching, we can mark the blueprints query's data as stale once we delete a blueprint, so a refetch happens automatically. To achieve that,  we can leverage the [enhanceEndpoints](https://redux-toolkit.js.org/rtk-query/api/created-api/code-splitting#enhanceendpoints)

For a reference, you can have a look at [this](https://github.com/RedHatInsights/image-builder-frontend/pull/1579/files#diff-4ba130e7ad33b29afaafd14838ac4737e4832315a9ad0dcd9aa99b300b8eb419), I've added cache management for refetching blueprint's images after building a blueprint.

In the future we can extend this behavior in favor optimistic updating